source/features/latest-tag-button.tsx

```diff
@@ -96,7 +96,7 @@ async function init(): Promise<false | void> {
 	});
 
 	const link = (
-		<a className="btn btn-sm btn-outline ml-2 flex-self-center" href={String(url)}>
+		<a className="btn btn-sm btn-outline ml-2 flex-self-center rgh-latest-tag-button" href={String(url)}>
```

Comment:
Again, changes in this file are not really related to this PR (sorry if it bothers😅). I added the class into the wrong position in the previous PR (https://github.com/sindresorhus/refined-github/pull/3497/files#r478513136), and I don't feel like make a separate PR for this trivial change.